pkg/controllers/termination/eviction.go

```diff
@@ -73,7 +73,7 @@ func (e *EvictionQueue) Start(ctx context.Context) {
 		nn := item.(types.NamespacedName)
 		// Evict pod
 		if e.evict(ctx, nn) {
-			logging.FromContext(ctx).Debugf("Evicted pod %s", nn.String())
+			logging.FromContext(ctx).Debug("Evicted pod %s", nn)
```

Comment:
This is gonna print poorly w/o Debugf

Reply:
Does it make sense to move this log line into the final line inside `e.evict(...)`. That way we can take advantage of the `With()` call on the logger.

Reply:
Took care of that in the latest push.  Also removed logs for the 404 condition.